src/plugins/foxPage/hooks/useOtherOpportunities.ts

```diff
@@ -21,7 +21,7 @@ import { OpportunityTypes } from '../FoxCommon'
 export const useOtherOpportunities = (assetId: AssetId) => {
   const highestFarmingBalanceAccountIdFilter = useMemo(
     () => ({
-      stakingId: foxEthStakingAssetIdV7 as StakingId,
+      stakingId: foxEthStakingAssetIdV8 as StakingId,
```

Comment:
That would theoretically rug the FOX page... except we don't have explicit routes to it anymore. Will remove it as a follow-up 🙏🏽

Reply:
thanks @gomesalexandre